dev/deploy-to-container/settings_local.py

```diff
@@ -64,6 +64,7 @@
 BIBXML_BASE_PATH = '/assets/ietfdata/derived/bibxml'
 IDSUBMIT_REPOSITORY_PATH = INTERNET_DRAFT_PATH
 FTP_DIR = '/assets/ftp'
+TMP_DIR = '/assets/tmp'
```

Comment:
Sort of a bike shed, but this is really a special purpose path. Calling it `TMP_DIR` might tempt someone to use it for random temp files when they'd be better off using a path not served over nfs. Maybe `NFS_TEST_DIR` (or `NFS_TMP_DIR` to at least emphasize its nature)